.github/workflows/runtime_upgrade_test.yml

```diff
@@ -17,7 +17,7 @@ env:
   AWS_IMAGE_SEARCH_PATTERN: ubuntu/images/hvm-ssd/ubuntu-focal-20.04-amd64-server-*
   AWS_IMAGE_SEARCH_OWNERS: '["099720109477"]'
   POLKADOT_BINARY: https://github.com/paritytech/polkadot/releases/download/v0.9.38/polkadot
```

Comment:
Switch to [v0.9.39-1](https://github.com/paritytech/polkadot/releases/tag/v0.9.39-1)?

Reply:
good catches. i thought it's not important to bump pdot since we just recently did it, but with a critical fix already deployed to pdot and kusama it's definitely needed